src/server/game/Entities/Object/ObjectPosSelector.h

```diff
@@ -136,7 +136,7 @@ struct ObjectPosSelector
     UsedPosList::value_type const* nextUsedPos(UsedPosType uptype);
 
     // angle from used pos to next possible free pos
-    float GetAngle(UsedPos const& usedPos) const { return acos(m_dist / (usedPos.dist + usedPos.size + m_size)); }
+    float GetAngle(UsedPos const& usedPos) const { return std::cos(m_dist / (usedPos.dist + usedPos.size + m_size)); }
```

Comment:
```suggestion
    float GetAngle(UsedPos const& usedPos) const { return std::acos(m_dist / (usedPos.dist + usedPos.size + m_size)); }
```

Reply:
that's odd as fuck, cuz I ran auto script for it to check explicitly for "cos" replace to "std::cos", where O.O